### DISCUSSION TITLE
Streaming video from a usb device

### DISCUSSION THREAD

**Discussion by ercanhocaninyeri** [zauberzeug/nicegui#1027] [Q&A] [answered]
Do you think that it is possible to display live video from a usb camera (e.g. simple webcam, gopro, or stereo camera) in a Nicegui frame?

**Comment by rodja** [comment, marked as answer]
Sure. Have a look at our [OpenCV example](https://github.com/zauberzeug/nicegui/blob/main/examples/opencv_webcam/main.py).
